gdnative/src/property.rs

```diff
@@ -70,11 +70,11 @@ bitflags! {
         const STORE_IF_NONZERO = 512;
         const STORE_IF_NONONE = 1024;
         const NO_INSTANCE_STATE = 2048;
-        const RESTART_IF_CHANGED = 4096;
```

Comment:
In this case I'd rather keep the constant as it is rather than using hexadecimals because this is how the constant is written in godot's source code, and keeping the same notation is less error prone and easier ot double check in doubt.

What would be even better actually is to use the constants exposed from the sys crate, for example:

```rust
use sys::godot_property_usage_flags::*;
bitflags! {
    pub struct PropertyUsage: u32 {
        const STORAGE = GODOT_PROPERTY_USAGE_STORAGE;
        // etc.
```

That way it's hard to get it wrong and in the unlikely event that the constants change from under us in godot, we don't need to do anything.

Reply:
I get an error with the structure type:
> expected u32, found enum `sys::godot_property_usage_flags`

Reply:
Oh right, you'd need to do:

```rust
use sys::godot_property_usage_flags::*;
bitflags! {
    pub struct PropertyUsage: u32 {
        const STORAGE = GODOT_PROPERTY_USAGE_STORAGE as u32;
        // etc.
```